package.json

```diff
@@ -75,6 +75,19 @@
       "max-len": [
         2,
         80
+      ],
+      "import/max-dependencies": [
+        "error", {
+          "max": 16
+        }
+      ],
+      "semi": [
+        "error",
+        "never"
+      ],
+      "object-curly-spacing": [
+        "error",
+        "never"
```

Comment:
Are these changes necessary?

Reply:
The tests do not pass without these changes.  I took a look at the [config repo](https://github.com/kentcdodds/eslint-config-kentcdodds) and did not see the cases for this.  Therefore when the `npm start validate` ran the files were handled according to the rules but two failed to occur.  The removal of a `;` and the spacing around an object `import { something } from 'something` vs `import {something} from 'something'`.  The test required tha latter to pass.